frozen-abi/src/abi_example.rs

```diff
@@ -431,6 +431,28 @@ impl<
     }
 }
 
+#[cfg(not(target_os = "solana"))]
+impl<T: Clone + std::cmp::Eq + std::hash::Hash + AbiExample, S: Clone + AbiExample> AbiExample
```

Comment:
given the other changes, think we can drop adding this.

Reply:
done by dropping the commit from the pr.